local/config/finna/config.ini.sample

```diff
@@ -801,9 +801,11 @@ recordMap = openlayers
 ;mapLabels = file:geosearch_test_lookup.txt
 ;displayCoords = true
 
-; Enables zoom for images in the image popup, only applies to records using the large image
-; layout. Disabled by default.
-; enableImagePopupZoom = true
+; Enables zoom for images in the image popup, only applies to 
+; record formats specified in zoomFormats key. Disabled by default.
+;enableImagePopupZoom = true
+; Formats which are allowed to use the zoom property, separated by :
```

Comment:
Hämää tuo kaksoispiste lopussa, voisi laittaa vaikka `A colon-separated list of formats that are enabled for image zoom function`